### DISCUSSION TITLE
With Sentry Example Update

### DISCUSSION THREAD

**Discussion by kmdev1** [vercel/next.js#17230] [Help]
Hey there, 

I was just wondering if it would be worth updating the [with-sentry](https://github.com/vercel/next.js/tree/canary/examples/with-sentry) example to use [`@sentry/react`](https://docs.sentry.io/platforms/javascript/guides/react/) rather than `@sentry/browser` and whether anyone knows if it's supported fully?

**Comment by m-bp** [comment]
Sentry has recently released [`@sentry/nextjs`](https://docs.sentry.io/platforms/javascript/guides/nextjs/). The examples should be updated to use that instead.

**Reply by ivan-kleshnin** [reply]
https://docs.sentry.io/platforms/javascript/guides/nextjs/

It uses experimental and highly unstable NextJS plugin... Not sure if it's a good idea to change current example yet 🤔
